src/main/java/org/folio/qm/converter/AbstractMarcDtoConverter.java

```diff
@@ -120,7 +120,7 @@ private Object processControlField(ControlField controlField, String leader) {
       0);
   }
 
-  private Map<String, Object> splitGeneralInformationControlField(String content, String leader) {
+  protected Map<String, Object> splitGeneralInformationControlField(String content, String leader) {
```

Comment:
Maybe will be better to make this method abstract because this implementation in the abstract class is suited only for bib records.